src/QueryM.purs

```diff
@@ -71,23 +126,23 @@ import Wallet (Wallet(Nami), NamiWallet, NamiConnection)
 --------------------------------------------------------------------------------
 -- Websocket Basics
 --------------------------------------------------------------------------------
-foreign import _mkWebSocket :: Url -> Effect WebSocket
+foreign import _mkWebSocket :: Url -> Effect JsWebSocket
 
-foreign import _onWsConnect :: WebSocket -> (Effect Unit) -> Effect Unit
+foreign import _onWsConnect :: JsWebSocket -> (Effect Unit) -> Effect Unit
 
-foreign import _onWsMessage :: WebSocket -> (String -> Effect Unit) -> Effect Unit
+foreign import _onWsMessage :: JsWebSocket -> (String -> Effect Unit) -> Effect Unit
 
-foreign import _onWsError :: WebSocket -> (String -> Effect Unit) -> Effect Unit
+foreign import _onWsError :: JsWebSocket -> (String -> Effect Unit) -> Effect Unit
 
-foreign import _wsSend :: WebSocket -> String -> Effect Unit
+foreign import _wsSend :: JsWebSocket -> String -> Effect Unit
 
-foreign import _wsClose :: WebSocket -> Effect Unit
+foreign import _wsClose :: JsWebSocket -> Effect Unit
 
-foreign import _stringify :: forall a. a -> Effect String
+foreign import _stringify :: forall (a :: Type). a -> Effect String
 
-foreign import _wsWatch :: WebSocket -> Effect Unit -> Effect Unit
+foreign import _wsWatch :: JsWebSocket -> Effect Unit -> Effect Unit
 
-data WebSocket
+data JsWebSocket
```

Comment:
Since you're making a change here already, I think it might make sense to turn this into a `foreign import data` since we use that everywhere else